sdk/spring/spring-cloud-azure-autoconfigure/src/main/java/com/azure/spring/cloud/autoconfigure/aad/properties/AadAuthenticationProperties.java

```diff
@@ -658,6 +677,12 @@ private void validateAuthorizationGrantType(String registrationId, AadAuthorizat
                 + AZURE_CLIENT_REGISTRATION_ID
                 + ".authorization-grant-type must be configured to 'authorization_code'.");
         }
+        //TODO: moary expose a way to let uses enable the ON_BEHALF_OF provider
```

Comment:
do we need this

Reply:
We'll replace it with jwt_bearer, but we will keep the ON_BEHALF_OF, and the implementation will only use JWT_BEARER.